src/common/storage.ts

```diff
@@ -61,3 +61,11 @@ export function deleteTabForRequest(storageKey: StorageKey, request: string) {
   const key = getKeyForRequest(storageKey, request);
   localStorage.removeItem(key);
 }
+
+export function persistOptional(storageKey: string, value?: string) {
```

Comment:
In https://github.com/hirosystems/stacks-wallet-web/pull/2061 I've renamed this method to `setToLocalstorageIfDefined` as this is a really bad name. Perhaps we can rename here instead.

I think we can also remove the else (even more confusing that it removes that value).